src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -29,6 +29,26 @@ set IMAGES = {
 }
 -%}
 jobs:
+  license:
+    runs-on: ubuntu-latest
+    name: Check licenses
+    container: {{ IMAGES["odoo"][odoo_version] }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install addons and dependencies
+        run: oca_install_addons
+      - name: Check licenses
+        run: manifestoo -d . check-licenses
+  dev-status:
+    runs-on: ubuntu-latest
+    name: Check development status
+    container: {{ IMAGES["odoo"][odoo_version] }}
+    steps:
```

Comment:
Would it be good to make these optional for some observation time? 🤔 

Just to avoid possible flood of ❌ in CIs.

```suggestion
  license:
    runs-on: ubuntu-latest
    name: Check licenses
    container: {{ IMAGES["odoo"][odoo_version] }}
    continue-on-error: true
    steps:
      - uses: actions/checkout@v2
      - name: Install addons and dependencies
        run: oca_install_addons
      - name: Check licenses
        run: manifestoo -d . check-licenses
  dev-status:
    runs-on: ubuntu-latest
    name: Check development status
    container: {{ IMAGES["odoo"][odoo_version] }}
    continue-on-error: true
    steps:
```

Reply:
Last time I checked, continue-on-error did show green status, so people will not even pay attention. It's a limitation of GitHub actions that they don't have a warning status.

Reply:
Yes.

Well, that was to avoid too many broken builds, but that's just an hypothesis. We can release an alpha template and check on some selected repos.